src/api/InfluxDB.cpp

```diff
@@ -60,6 +60,8 @@ vz::api::InfluxDB::InfluxDB(const Channel::Ptr &ch, const std::list<Option> &pOp
 	try {
 		_token = optlist.lookup_string(pOptions, "token");
 		print(log_finest, "api InfluxDB using login Token: %s", ch->name(), _token.c_str());
+		_token = "Authorization: Token " + _token;
+		_token_header = curl_slist_append(_token_header, _token.c_str());
 	} catch (vz::OptionNotFoundException &e) {
 		print(log_finest, "api InfluxDB no Token set", ch->name());
```

Comment:
_token_header should be init to zero here

Reply:
(or better in the contructor earlier (before the curl_slist_append)

Reply:
modern C, stuff is zero-initialized by default.

Reply:
fixed